Calculator/Calculator/CalculatorItemQueue.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  CalculatorItemQueue.swift
+//  Calculator
+//
+//  Created by Jaehun Lee on 2/6/24.
+//
+
+protocol CalculateItem {
+    
+}
+
+protocol QueueProtocol {
```

Comment:
```suggestion
protocol Queueable {
```

취향인데 저는 Protocol은 able 붙이는게 좋더라구요 ㅋㅋ